sudoku/sudoku.rb

```diff
@@ -0,0 +1,88 @@
+class Sudoku
+  def initialize(board)
+    @board = board
+  end
+
+  def done_or_not?
+    check_rows? && check_columns? && check_cells?
+  end
+
+  private
+
+  def check?(rows)
+    rows.all? {|row| row.sort.join('') == Array(1..9).join('')}
+  end
+
+  def check_rows?
```

Comment:
probably these small methods are not necessary since they are quite readable in one line